src/styles/base/mixins/_step.scss

```diff
@@ -1,20 +1,5 @@
 @mixin step($attribute, $value, $i: false) {
   $important: if($i, "!important", "");
-  $calc-sm: calc-step($value, $mc-step-decay-sm);
-  $calc-md: calc-step($value, $mc-step-decay-md);
-  $calc-lg: calc-step($value, $mc-step-decay-lg);
 
-  #{$attribute}: $calc-sm * 1rem #{$important};
-
-  @if $calc-md != $calc-sm {
-    @media (min-width: $mc-bp-md) {
-      #{$attribute}: $calc-md * 1rem #{$important};
-    }
-  }
-
-  @if $calc-lg != $calc-sm {
-    @media (min-width: $mc-bp-lg) {
-      #{$attribute}: $calc-lg * 1rem #{$important};
-    }
-  }
+  #{$attribute}: var(--mc-scale-#{$value}) #{$important};
```

Comment:
If this is being simplified, that means all existing implementations of the step helper will need to be updated to use css vars in the main masterclass repo, right?

Reply:
this mixin is those instances. it's effectively retrofitting all of them with the new way so we don't have to replace them all

Reply:
duh.